coralproject/talk
# Server Graph Plugins

## What does this PR do?

- Adds initial support for the graph to have injectable plugins

## How do I test this PR?

- Review the documentation on plugins `PLUGINS.md`
- Attempt to create a plugin
- Check `/admin/docs`, and `/api/v1/graph/iql` to see if new plugins were added
